### PR TITLE
Add support for PostgreSQL 16

### PR DESCRIPTION
**Pull request by svoop**
Works for me. 😄

### CODE REVIEW

File: Formula/postgresql@16.rb [hunk 1]
```diff
@@ -0,0 +1,139 @@
+class PostgresqlAT15 < Formula
```
Comment (ozeias) [inline Formula/postgresql@16.rb:1]:
```suggestion
class PostgresqlAT16 < Formula
```

File: Formula/postgresql@16.rb [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  desc "Object-relational database system"
+  homepage "https://www.postgresql.org/"
+  url "https://ftp.postgresql.org/pub/source/v15.6/postgresql-15.6.tar.bz2"
```
Comment (ozeias) [inline Formula/postgresql@16.rb:4]: Still installing PostgreSQL 15, it should be 16.

**Review by ozeias — CHANGES_REQUESTED** [review]

File: Formula/postgresql@16.rb (hunk 1 above)
Comment (svoop) [reply Formula/postgresql@16.rb:1]: Aw, shoot, my bad. I have another extension in my forked tap and to keep it simple for our team, we'll install off that fork. Should have taken more care when copying over the files. I'll force push the right one.

**Comment by svoop** [comment]
Apparently, it doesn't pick up forced fixes. I'll just create a new PR instead, sorry for the noise.

**Pull request closed without merging**